frontend/pages/tools.vue

```diff
@@ -39,6 +39,7 @@
         <div class="divide-y divide-gray-300 border-t border-gray-300 px-6 pb-3">
           <DetailAction @action="modals.import = true">
             <template #title> {{ $t("tools.import_export_set.import") }} </template>
+            <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Not sure whether we should switch to using templates see (https://stackoverflow.com/questions/70751297/i18n-translate-a-sentence-that-one-of-words-is-bold).

Reply:
I didn't see the template thing in the documentation I was reading https://i18n.nuxtjs.org/docs/getting-started however, if it works, I think it would make sense to switch.

Reply:
Can't seem to get this to work so I'll leave it for now, not too big of an issue anyway.